custom_components/daikin_onecta/climate.py

```diff
@@ -247,115 +217,76 @@ def get_supported_features(self):
         supported_features = 0
         if hasattr(ClimateEntityFeature, "TURN_OFF"):
             supported_features = ClimateEntityFeature.TURN_OFF | ClimateEntityFeature.TURN_ON
-        setpointdict = self.setpoint()
-        cc = self.climateControl()
-        if setpointdict is not None and setpointdict["settable"] is True:
+        setpoint_dict = self.setpoint()
+        cc = self.climate_control()
+        if setpoint_dict is not None and setpoint_dict["settable"] is True:
             supported_features |= ClimateEntityFeature.TARGET_TEMPERATURE
         if len(self.get_preset_modes()) > 1:
             supported_features |= ClimateEntityFeature.PRESET_MODE
-        fanControl = cc.get("fanControl")
-        if fanControl is not None:
-            operationmode = cc["operationMode"]["value"]
-            if fanControl["value"]["operationModes"][operationmode].get("fanSpeed") is not None:
+        fan_control = cc.get("fanControl")
+        if fan_control is not None:
+            operation_mode = cc["operationMode"]["value"]
+            if fan_control["value"]["operationModes"][operation_mode].get("fanSpeed") is not None:
                 supported_features |= ClimateEntityFeature.FAN_MODE
-            if fanControl["value"]["operationModes"][operationmode].get("fanDirection") is not None:
+            if fan_control["value"]["operationModes"][operation_mode].get("fanDirection") is not None:
                 supported_features |= ClimateEntityFeature.SWING_MODE
 
-        _LOGGER.info("Device '%s' supports features %s", self._device.name, supported_features)
-
         return supported_features
 
     @property
     def name(self):
-        device_name = self._device.name
-        cc = self.climateControl()
-        namepoint = cc.get("name")
-        if namepoint is not None:
-            device_name = namepoint["value"]
-        myname = self._setpoint[0].upper() + self._setpoint[1:]
-        readable = re.findall("[A-Z][^A-Z]*", myname)
-        return f"{device_name} {' '.join(readable)}"
+        return self._device.name
```

Comment:
As name I return the name as set in the onecta app, I think this breaks that. Also the user can rename the device in the onecta app easily, at that moment the names in HA should be updated

Reply:
Let me give you my example on why I changed that. I have a unit named Bedroom for example. With the current approach, on HA, it will be named climate.bedroom_room_temperature. With my change it simply become climate.bedroom, which is cleaner in my opinion.

Reply:
There are daikin devices which have multiple setpoints, for example leaving water and room temperature, some have even more